model/feature/flag.go

```diff
@@ -71,6 +71,38 @@ func (f *Flags) UnmarshalJSON(bytes []byte) error {
 	return nil
 }
 
+func (f *Flags) GetList(name string) ([]interface{}, error) {
+	if f.M[name] == nil {
+		return []interface{}{}, nil
+	}
+
+	value, ok := f.M[name].(map[string]interface{})
+	if !ok {
+		return nil, fmt.Errorf("Flag %s is not a list flag", name)
+	}
+
+	list, ok := value["list"].([]interface{})
```

Comment:
I'm surprised by the format here. I would have expected `f.M[name]` to be `[]interface{}`, not having to look at a `list` thing. Are you sure about the format?

Reply:
As discussed, this is the format used for list flags.